benchmarks/templates/benchmarks/leaderboard/leaderboard-table.html

```diff
@@ -0,0 +1,152 @@
+{% if models %}
+{% if has_user %}
+
+{% load static %}
+
+<form>
+{% endif %}
+
+<banner>
+  <div class="leaderboard-table-banner-icon gradient">
+      <img src="{% static "/benchmarks/img/leaderboard_header_icon.png" %}" alt="Brain-Score Logo">
+  </div>
+  <div class="leaderboard-table-banner box gradient"><h3>Leaderboard</h3></div>
+</banner>
+<div class="filters">
+  <select class="benchmark-sort">
+    <option value="" disabled selected>Sort by Benchmarks</option>
```

Comment:
So this is great, it looks like it reorders the models like expected when sorting. If it is not too complicated, it would be cool to have whatever selected benchmark to sort by becomes the leftmost root in the table.

Reply:
Will be addressed in a follow-up pull